src/proto/media/partition.rs

```diff
@@ -108,6 +108,17 @@ pub struct GptPartitionEntry {
     pub partition_name: [Char16; 36],
 }
 
+impl GptPartitionEntry {
+    /// Get the number of blocks in the partition. Returns None if the
```

Comment:
Just a minor typographic suggestion:
```suggestion
    /// Get the number of blocks in the partition. Returns `None` if the
```